runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -103,19 +103,19 @@ pub trait WeightInfo {
     fn create_proposal_veto_proposal(t: u32, d: u32) -> Weight;
 }
 
-type WeightInfoCodex<T> = <T as Trait>::WeightInfo;
+type WeightInfoCodex<T> = <T as Config>::WeightInfo;
 
-/// 'Proposals codex' substrate module Trait
-pub trait Trait:
-    frame_system::Trait
-    + proposals_engine::Trait
-    + proposals_discussion::Trait
-    + common::membership::Trait
-    + staking::Trait
-    + proposals_engine::Trait
+/// 'Proposals codex' substrate module Config
+pub trait Config:
+    frame_system::Config
+    + proposals_engine::Config
+    + proposals_discussion::Config
+    + common::membership::Config
+    + staking::Config
+    + proposals_engine::Config
```

Comment:
Why do we have a duplicate here?

Reply:
Duplicate removed https://github.com/Joystream/joystream/pull/2558/commits/ff16b9a952475e890b41e236367174e275459e9d